CMSgov/mint-app
# [EASI-2421] IT Lead to roles drop down

# EASI-2421

## Changes and Description

- BE - add `IT_LEAD` enum to `TeamRole` enum array
- FE - add translation and selection for new collaborator role option




## How to test this change

1. Create a new model plan and click through till the `Add member` page
2. Ensure `Team member role` has `IT Lead`
3. Ensure the role is properly displaying `IT Lead`
4. Navigate to Read Only's Team page and ensure IT Lead is showing properly



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
